From e84bf343ab7141854c6174b9a062d652de8a461a Mon Sep 17 00:00:00 2001
From: razorloves <razorloves@gmail.com>
Date: Thu, 9 Nov 2017 07:00:59 -0600
Subject: [PATCH 23/26] config: Remove 7z lib7z unrar unzip zip makefile
 entries

Appear to be no longer used.  Originally added for cmfilemanager.

Change-Id: I0cb3904d3a14c1b15fc7d43f8bb51d2812791409
---
 config/common.mk | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 81015818..c7296628 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -172,7 +172,6 @@ PRODUCT_PACKAGES += \
 
 # Extra tools in Lineage
 PRODUCT_PACKAGES += \
-    7z \
     awk \
     bash \
     bzip2 \
@@ -180,7 +179,6 @@ PRODUCT_PACKAGES += \
     fsck.ntfs \
     gdbserver \
     htop \
-    lib7z \
     libsepol \
     micro_bench \
     mke2fs \
@@ -192,11 +190,8 @@ PRODUCT_PACKAGES += \
     sqlite3 \
     strace \
     tune2fs \
-    unrar \
-    unzip \
     vim \
-    wget \
-    zip
+    wget
 
 # Charger
 PRODUCT_PACKAGES += \
-- 
2.17.1

